frontend/src/components/commons/Constants.jsx

```diff
@@ -0,0 +1,2 @@
+export const HEADER_HEIGHT = 130;
```

Comment:
These aren't really common - they are eMIB specific and JSX is the file type for a react component. I'd change this to `/eMIB/constants.js` and add some comments in here explaining what these describe and why they are necessary.

Reply:
Another option would be to define each constant within the component it refers to, so one in footer, and one in wherever the nav is defined. That's up to you.

Reply:
I agree with @caleybrock on that. These are specific to eMIB. I think that having a constants file would simplify things in case we have to resize these windows in the future.

Reply:
Refactored in: f54138c